config/nut/nut.inc

```diff
@@ -333,7 +326,7 @@ EOD;
 EOD;
 
 		$start = <<<EOD
-if [ `pgrep {$driver} | wc -l` != 0 ]; then
+if [ `/bin/pgrep {$driver} | /usr/bin/wc -l` != 0 ]; then
```

Comment:
Since you are touching this, you could make this simpler using pgrep return code instead of wc:
```
if /bin/pgrep -q {$driver} 2>/dev/null; then
    {$stop}
fi
```